src/main/webapp/assets/js/index.js

```diff
@@ -164,13 +196,7 @@ function homePageListingTemplate(business) {
           <p class="categories">${product.categories}</p>
           <p class="price">Price: From \$${product.minPrice}</p>
           <p>${description}...</p>
-          <p class="rating">Rating: ${product.aggregatedRating}</p>
-          <span>
-            <button>
-              <i class="fa fa-cart-arrow-down"></i>
-              <a href=${product.contactUrl}>Contact Business</a>
```

Comment:
Why is this part removed tho?

Reply:
i derped and i thought this linked to to the product itself. Now i realised it's the specific contactUrl. I'll put it back in